docker/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN set -ex \
   && corepack prepare pnpm@latest --activate \
   && pnpm set progress=false \
   && pnpm config set depth 0 \
-  && pnpm install --ignore-scripts --frozen-lockfile \
+  && pnpm install --force --ignore-scripts --frozen-lockfile \
```

Comment:
Why forcing? The docker image build successfully without it.

Reply:
I noticed on some subsequent rebuilds that the docker image build was hanging on the `pnpm install` step.

I just tried reproducing the issue though and I am not able to. I'll take the `--force` flag out and open an issue if it comes up again in the future.

Reply:
Actually, I ran the image build again and was just able to reproduce it:
<img width="630" alt="image" src="https://github.com/user-attachments/assets/c941d863-0a4d-462d-9998-132e256e8e3e" />

It looks like pnpm is waiting for interactive input on the step:

> → ? The modules directories will be removed and reinstalled from scratch. Proceed? (Y/n) ‣ true

When I use the `--force` flag it automatically proceeds on this step.

Reply:
Using --force on CLI is hardly a proper fix at any command usage, unless you know what you are doing. A proper fix was pushed to the main branch yesterday.

Reply:
Thanks, I'll merge that change into this branch.